spotifycli/spotifycli.py

```diff
@@ -49,6 +54,25 @@ def main():
         control_volume("-5%")
 
 
+def start_shell():
+    while(True):
+        try:
+            command = input('spotify > ')
```

Comment:
Please, change `input(...)` function to `raw_input(...)`. It should fix the following error:
```
File "<string>", line 1, in <module>
```

Reply:
Oops sorry! I am not a master in python :)

Reply:
Me too. Just found after quick research ;).